templates/workloads/services/lb-web/cf.yml

```diff
@@ -246,6 +246,14 @@ Resources:
         - TargetGroupArn: !Ref TargetGroup
           Type: forward
       Conditions:
+      {{- if .AllowedSourceIps}}
+        - Field: 'source-ip'
+          SourceIpConfig:
+            Values:
+            {{- range $sourceIP := .AllowedSourceIps}}
+            - {{$sourceIP}}
+            {{- end}}
+            {{- end}}
```

Comment:
Could you make the last `end` to be aligned with the `if` above?

Reply:
Yes! Thanks!